src/sinks/elasticsearch.rs

```diff
@@ -64,13 +67,223 @@ pub struct ElasticSearchConfig {
 
     pub aws: Option<RegionOrEndpoint>,
     pub tls: Option<TlsOptions>,
+    // Deprecated, use normal.bulk_action instead
     pub bulk_action: Option<String>,
```

Comment:
@jdrouet I'm just realizing that I don't think these new `normal.bulk_action` and `normal.index` options were documented.